engine/access/rpc/backend/script_executor.go

```diff
@@ -85,6 +90,18 @@ func (s *ScriptExecutor) ExecuteAtBlockHeight(ctx context.Context, script []byte
 		return nil, err
 	}
 
+	// Version control feature could be disabled by cmd argument "--version-control-enabled". In such a case, ignore related functionality.
+	if s.versionControl != nil {
```

Comment:
can you move this logic into `checkDataAvailable` so it's used for all methods (including get account). it would make sense to rename the method to something like `checkHeight`